### ISSUE TITLE
dhcpy6d prints `Error: table meta already exists` on startup

### ISSUE THREAD

**Issue by FelixJacobi** [HenriWahl/dhcpy6d#45]
Hi,

~~~
Sep 22 13:47:16 iserv.felix.mein-iserv.de dhcpy6d[4187198]: Query: CREATE TABLE meta (item_key varchar(255) NOT NULL,                                  item_value varchar(255) NOT NULL, PRIMARY KEY (item_key))
Sep 22 13:47:16 iserv.felix.mein-iserv.de dhcpy6d[4187198]: Error: table meta already exists
~~~

I configured sqlite as volatile database storage:

~~~
felix.mein-iserv.de ~ # grep volatile /etc/dhcpy6d.conf
store_volatile = sqlite
store_sqlite_volatile = /var/lib/dhcpy6d/volatile.sqlite
~~~

After upgrading to Debian Bullseye where a new dhcpy6d version is included, this message is now printed on startup. There was already an existing SQLite database created by dhcpy6d many years ago in place.

The message is printed several times on startup and then once a ~minute again.

Thank you for your time.

Best regards,
Felix

**Comment by HenriWahl** [comment]
Can you send me the sqlite file for testing it?

Looks like the attempt to create the meta table should not block if the table already exists.

**Comment by FelixJacobi** [comment]
> Can you send me the sqlite file for testing it?
> 
> Looks like the attempt to create the meta table should not block if the table already exists.

Send via email. Thanks so far.

**Comment by HenriWahl** [comment]
fixed with v1.0.6
